Loosen requirement on HTTP resources (#220)
ietf-wg-aipref/drafts: draft-ietf-aipref-attach.md

### Embedded Preferences

Embedding preferences is expected to be an effective means
of associating preferences with assets,

> because it ensures that metadata travels with the asset.

Reviewer: Remember #163.

Author: Yeah, it would be good to resolve that.  The goal here is to close the distance between the two documents first.  Then at least it is easier to do a search and replace if we conclude somehow.

> because it ensures that metadata travels with the asset.

Reviewer 2: ```suggestion
because it allows metadata to travel with the asset.
```

It doesn't ensure it because it can be stripped.

Informally, usage preferences apply to the content of a message,
not the resource.

Servers can use any preferences associated with a request
if the content of that request
is used to answer later requests.
For example,
attached preferences for a PUT request,
where the content that is used to answer subsequent GET requests,

> might be copied to the response to those GET requests.

Reviewer 2: This is old text, but how would I indicate to the server that I don't want it to train on this content but that I don't want it to propagate the preference?

Author: I don't think that's possible.  Worth an issue?

For this to happen,

> servers or resources need to be updated

Reviewer 2: ```suggestion
servers or resources would need to be updated
```
